packages/online-editor/it-tests/integration/DMNExpressionEditor.ts

```diff
@@ -23,49 +23,49 @@ describe("DMN Expression Editor Test", () => {
 
   it("Test New Expresssion editor - context", () => {
     // click Create new decision model button (new DMN)
-    cy.ouiaId("try-dmn-sample-button").click();
+    cy.ouia({ ouiaId: "try-dmn-sample-button" }).click();
```

Comment:
I think the semantics define ouaiId as a distinction of several components of the same ouiaType. In this case we're not certain what the ouiaType is, so this is prone to error. ouiaId does not need to be globally unique, it's just supposed to be unique in surrounding context - the actual component. So let's say we have a table, with multiple rows. I would consider it a valid state when each of these rows contain a button with ouiaId='buttonSubmit' - it is uniquely distinguished component within surrounding context (the table row).
But this does not look like a problem inflicted by these changes.
More in https://ouia.readthedocs.io/en/latest/